content/en/blog/2022-09-01-manage-kyverno-policies-as-ocirepositories/index.md

```diff
@@ -0,0 +1,213 @@
+
+> ⚠️ **Note:** You can read the RFC of this feature [here](https://github.com/fluxcd/flux2/tree/main/rfcs/0003-kubernetes-oci).
+
+Today’s blog post is all about a quick tour of this feature and will give you a real-world example of it to show you how you can leverage this feature to manage Kyverno policies as OCI Artifacts. It is worth saying that this topic has been discussed for a while in the Kyverno community, too. There is an ongoing [issue](https://github.com/kyverno/KDP/pull/19) about packaging and distributing Kyverno policies as OCI Artifacts through its CLI. Also, there is a chance to move that logic into Kyverno’s core.
+
+But for those who might not be familiar enough with OCI artifacts (including me), it’s worth explaining what the OCI Artifacts are before jumping into the details. OCI Artifacts gives you the power of storing and distributing other types of data (nearly anything), such as Kubernetes deployment files, [Helm Charts](https://helm.sh/), [and CNAB](https://cnab.io/), in addition to container images via OCI registries. And today, we’ll be using this feature for Kyverno policies. To be more precise, OCI Artifacts are not a new specification, format, or API. It just utilizes the existent [OCI manifest](https://github.com/opencontainers/image-spec/blob/master/manifest.md) and [OCI index](https://github.com/opencontainers/image-spec/blob/master/image-index.md) definitions. Hence, we can quickly start using the same client tooling, such as a crane, skopeo, etc., and distribute them using OCI registries, thanks to the [OCI distribution-spec](https://github.com/opencontainers/distribution-spec/). Because OCI Artifacts does not change anything related to the specs, it only expands them to give people (artifact authors) power to define their content types. It is more like a generic definition for determining what can be stored in an OCI registry and consumed by clients.
+
+The Flux CLI generates a single layer OCI image for storing things. As you can use some other tools to generate an OCI image with multiple layers in it, you can use the [Layer Selection](https://github.com/fluxcd/flux2/tree/main/rfcs/0003-kubernetes-oci#layer-selection) feature that Flux provides to select the layers you want to use in the OCI image. If the layer selector matches more than one layer, the first layer matching the specified media type will be used. Note that Flux requires that the OCI layer is compressed in the tar+gzip format.
+
+{{< imgproc twitter-ss Resize 400x >}}
+{{< /imgproc >}}
+
+{{< imgproc meme Resize 400x >}}
```

Comment:
Could be nice if the file was called `meme-featured` and it would turn up as the feature image of the blog.

Reply:
thank you so much for your help